pepper-apis/src/main/java/org/broadinstitute/ddp/db/UserDao.java

```diff
@@ -392,8 +392,12 @@ private User insertNewUser(Handle handle, String auth0ClientId, String auth0User
     /**
      * Returns user permissions for the given operator user guid.
      */
-    public UserPermissions queryUserPermissionsByOperatorGuidAndClientId(Handle handle, String operatorGuid,
-                                                                         String auth0ClientId) {
+    public UserPermissions queryUserPermissionsByOperatorGuidClientIdAndAuth0Domain(
```

Comment:
At some point, the `AndFooAndBar` starts to get a bit unwieldy.  This might be that point.  How about just `queryUserPermissions()`?

Reply:
Done.